src/bedrock/kvs/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+ADD_EXECUTABLE(flkvs server.cpp)
```

Comment:
Why do none of the other executables have the `${ZMQ_DEPENDENCIES}`? All of the files (`server.cpp`, `monitoring.cpp`, and `routing.cpp`) all import `socket_cache.h` and `zmq_util.h`.

Reply:
ZMQ code is compiled into the hash_ring library and everyone else links to it from there.

The header files for ZMQ are not auto-generated hence don't change as a result of building, so they probably don't need to be compiler dependencies any more than system libraries need to be compiler dependencies.